authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -538,6 +539,15 @@ func getNameSpaceScope(req *authzv1.SubjectAccessReviewSpec, useNamespaceResourc
 	return false, namespace
 }
 
+func getManagedNameSpaceScope(req *authzv1.SubjectAccessReviewSpec) (bool, string) {
```

Comment:
nit `getManagedNamespaceScope`

Reply:
I tried to keep it consistent with getNameSpaceScope method (used above)